Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -3992,22 +3992,22 @@ void WebPageProxy::processNextQueuedMouseEvent()
     ASSERT(!internals().mouseEventQueue.isEmpty());
     m_deferredMouseEvents = 0;
 
-    const NativeWebMouseEvent& event = internals().mouseEventQueue.first();
+    const CheckedRef event = internals().mouseEventQueue.first();
```

Comment:
It seems debug builds are crashing on this or the next CheckedRef in this method. @cdumez I can remove these changes, but is there something even better I could be doing?